drv/stm32h7-update-server/src/main.rs

```diff
@@ -388,6 +388,13 @@ impl idl::InOrderUpdateImpl for ServerImpl<'_> {
             version: HUBRIS_BUILD_VERSION,
         })
     }
+
+    fn clock_speed(
+        &mut self,
+        _: &RecvMessage,
+    ) -> Result<u32, RequestError<UpdateError>> {
+        Err(UpdateError::Unknown.into()) // Not implemented for the STM32
+    }
```

Comment:
Since this doesn't seem to be used currently can we just remove this API and add it later?

Reply:
I'll remove it.